azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -100,6 +103,13 @@ internal open class CallCompositeActivity : AppCompatActivity() {
             return
         }
 
+        store.dispatch(DeviceConfigurationAction.IsTableChanged(isTablet(this)))
```

Comment:
Action, so should be verbiage (I.e. DeviceConfigruation.ToggleTabletMode. I.e. where "Toggle" is the action. 

Also Table->Tablet